webpack.config.js

```diff
@@ -39,6 +39,10 @@ module.exports = {
                 test: /\.xml$/i,
                 use: ['text-loader'],
             },
+            {
+              test: /\.(eot|svg|ttf|woff|woff2)$/,
+              loader: 'file-loader?name=/fonts/[name].[ext]'
```

Comment:
This is actually unused because we force file-loader at every import.
@Goaman This is your area of expertise... If you find a better solution please be my guest ;-)

Reply:
True that for now, we don't need them as we do not import those format without "inline-import".
We can investigate in the future for having those files automatically configured in one place. But right now it seems that the css-loader does not inform webpack properly about the files it tries to access. Thus webpack do not try to load them.

Reply:
Ok so should I remove these lines for the time being?

Reply:
@Goaman ?

Reply:
Yep, fine with me.